node/lib/assets.js

```diff
@@ -112,36 +122,27 @@ function init (options, directory, locals) {
 
 				res.render = function (template, templateData) {
 
-					let cssVariant = templateData.cssVariant || res.locals.cssVariant;
-					cssVariant = cssVariant ? `-${cssVariant}` : '';
-
+					// Add standard n-ui stylesheets
+					res.locals.cssBundles.inline.unshift('head-n-ui-core');
+					res.locals.cssBundles.lazy.unshift('n-ui-core');
 
-					// define which css to output in the critical path
-					if (options.withHeadCss) {
-						// variants of head-n-ui-core no longer exist, but the app may not necessarily
-						// have successfully fetched head-n-ui-core from network, so fallback to the variant
-						// file while trying it out
-						if ('head-n-ui-core' in stylesheets) {
-							res.locals.criticalCss.push(stylesheets['head-n-ui-core'])
-						} else if (`head${cssVariant}-n-ui-core` in stylesheets) {
-							res.locals.criticalCss.push(stylesheets[`head${cssVariant}-n-ui-core`])
+					res.locals.cssBundles.inline = res.locals.cssBundles.inline.reduce((str, name) => {
+						if (!stylesheets[name]) {
+							throw `Stylesheet ${name} does not exist`;
 						}
-						if (`head${cssVariant}` in stylesheets && stylesheets[`head${cssVariant}`].length) {
-							res.locals.criticalCss.push(stylesheets[`head${cssVariant}`]);
-						}
-					}
+						return str + stylesheets[name];
+					}, '');
 
-					res.locals.cssBundles.push({
-						path: hasher.get(`main${cssVariant}.css`),
-						isMain: true,
-						isLazy: options.withHeadCss
-					});
+					res.locals.cssBundles.lazy = res.locals.cssBundles.lazy.map(getStylesheetPath);
```

Comment:
all these things running over both sets of styles... should abstract out into a function